src/components/pages/Login/index.js

```diff
@@ -1,22 +1,26 @@
-import { Link } from "react-router-dom";
+
 import Button from "../../atoms/Button";
 import Input from "../../atoms/Input";
 import styles from './style.module.css';
 
 function LoginComponent({ onTextChange, onSignIn }) {
+  
   return (
-    <div>
+   
+    <div className={styles.container}>
       <div className={styles.flex}>
-        Email: <Input onChange={onTextChange('email')} />
-        Password: <Input onChange={onTextChange('password')} />
+        <Input  placeholder={"Name..."} onChange={onTextChange('name')} required />
```

Comment:
Don't need to pass the value in brackets, cause they are simple string, e.g `placeholder='Name'` and also, always use single quote for string values